shell/agents/Microsoft.Azure.Agent/Telemetry.cs

```diff
@@ -13,45 +14,15 @@ public class AzTrace
     /// </summary>
     internal static string InstallationId { get; private set; }
 
+    /// <summary>
+    /// OS platform the application is running on.
+    /// </summary>
+    internal static string Platform { get; private set; }
```

Comment:
Using `Platform` instead of `OSPlatform` here to avoid duplications with `System.Runtime.InteropServices.OSPlatform`.